query_test.go

```diff
@@ -64,7 +64,7 @@ func TestScalarCompare(t *testing.T) {
 			name:     "testing unknown operator",
 			x:        big.NewInt(0),
 			y:        big.NewInt(1),
-			operator: 4, // unknown operator.
+			operator: 6, // unknown operator.
```

Comment:
use 5.
Because 5 it is first failed status for Scalare comarator.

Reply:
fixed